content/documentation/staging/tutorial/06-secure/01-traffic-policies.adoc

```diff
@@ -0,0 +1,128 @@
+
+Istio aggregates in the telemetry the ServiceAccount information used by the workloads in the communication, and this information can be used to define authorization policies with deny and allow actions for a particular live traffic communication status.
+
+This step will show how we can define authorization policies for the *travel-agency* namespace in the Travel Demo application for all existing traffic in a given time.
+
+Once authorization policies are defined, a new workload will be rejected if it doesn't match the security rules defined.
+
+NOTE: *Step 1:* Undeploy the *loadbtester* workload from *travel-portal* namespace
+
+In this example we will use the *loadtester* workload as the "intruder" in our security rules.
+
+If we have followed the previous tutorial steps, we need to undeploy it from the system.
+
+[source,bash]
+----
+kubectl apply -f <(curl -L https://raw.githubusercontent.com/kiali/demos/master/travels/travel_loadtester.yaml) -n travel-portal
+----
+
+We should validate that telemetry has updated the *travel-portal* namespace and "Security" can be enabled in the Graph Display options.
+
+++++
+<a class="image-popup-fit-height" href="/images/tutorial/06-01-travel-portal-graph.png" title="Travel Portal Graph">
+    <img src="/images/tutorial/06-01-travel-portal-graph.png" style="display:block;margin: 0 auto;" />
+</a>
+++++
+
+NOTE: *Step 2:* Create Authorization Policies for current traffic for *travel-agency* namespace
+
+Every workload in the cluster uses a link:https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/[Service Account, window="_blank"].
+
+*travels.uk*, *viaggi.it* and *voyages.fr* workloads use the default *cluster.local/ns/travel-portal/sa/default" ServiceAccount defined automatically per namespace.
+
+This information is propagated into the Istio Telemetry and Kiali can use it to define a set of AuthorizationPolicy rules using the "Create Traffic Policies" action located in the Overview page.
+
+++++
+<a class="image-popup-fit-height" href="/images/tutorial/06-01-create-traffic-policies.png" title="Create Traffic Policies">
+    <img src="/images/tutorial/06-01-create-traffic-policies.png" style="display:block;margin: 0 auto;" />
+</a>
+++++
+
+{nbsp} +
+This will generate a main DENY ALL rule to protect the whole namespace, and individual ALLOW rule per workload identified in the telemetry.
+
+++++
+<a class="image-popup-fit-height" href="/images/tutorial/06-01-travel-agency-authorization-policies.png" title="Travel Agency Authorization Policies">
+    <img src="/images/tutorial/06-01-travel-agency-authorization-policies.png" style="display:block;margin: 0 auto;" />
+</a>
+++++
+
+NOTE: *Step 3:* Deploy the *loadtester* portal in the *travel-portal* namespace
+
+The *loadtester* workload uses a different ServiceAccount then when it's deployed it won't comply the AuthorizationPolicy rules defined in the previous step.
```

Comment:
-->  If the *loadtester* uses a different ServiceAccount then, when it's deployed,  it won't comply with the...